### ISSUE TITLE
General Issue: Investigate CTCAE v6

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiral#1858]
### Background Information

Investigate 

https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm#ctc_60

**Comment by bms63** [comment]
Closing until v6 becomes a viable option

**Comment by bundfussr** [comment]
CTCAE v6 is now available at https://dctd.cancer.gov/research/ctep-trials/trial-development#ctcae-and-ctep-codes. Review and implementation is requested at Roche.

**Comment by manciniedoardo** [comment]
@millerg23 would you like to take this on?

**Comment by millerg23** [comment]
Yeah, I need to meet up with some Roche colleagues to review document. Can then make a start.

**Comment by manciniedoardo** [comment]
@millerg23 are you considering moving from xlsx to JSON for this?
